config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/studios", to: "studios#index"
+  get "/studios/:id", to: "studios#show"
+
+  get "/movies/:id", to: "movies#show"
+
+  post "/movies/:id/actors", to: "movie_actors#create" 
```

Comment:
```suggestion
  post "/movies_actors", to: "movie_actors#create" 
```
This revision to the path is more restful for the controller/action combo you have and is how I would've directed this.